ftw/publisher/core/tests/test_portlet_adapter.py

```diff
@@ -208,3 +208,27 @@ def test_portlets_with_settings(self):
             IPortletAssignmentSettings.providedBy(settings),
             'Portlet settings is not PortletAssignmentSettings but %s' % (
                 type(settings)))
+
+    def test_portlet_visibility_settings(self):
+        assignment = self.layer['right_portlets'].get('title2', False)
+        settings = {'visible': True}
+        IPortletAssignmentSettings(assignment).data = settings
+
+        # get the data
+        adapter = getAdapter(self.layer['folder1'], IDataCollector,
+                             name="portlet_data_adapter")
+        data = adapter.getData()
```

Comment:
Globi, you handle that already:
https://github.com/4teamwork/ftw.publisher.core/blob/master/ftw/publisher/core/utils.py#L131

I'm sure it works since i tested it with two `izug.organisation` buildouts and successfully published the portlet from one to the other.